projects/packages/forms/src/blocks/form-step/editor.scss

```diff
@@ -20,14 +20,15 @@
 	}
 
 	.jetpack-form-step__label {
-		background-color: var(--wp-components-color-background, #FFF);
-		color: var(--wp-components-color-foreground, #1e1e1e);
+		background-color: var(--wp-components-color-background, gb.$white);
+		color: var(--wp-components-color-foreground, gb.$gray-900);
 		font-size: var(--wp--preset--font-size--normal, 16px);
 		padding: 0 calc(var(--wp--style--block-gap, 1.5rem) * 0.5);
-		border-radius: var(--wp--preset--spacing--20, 4px);
-		font-weight: 600;
+		border-radius: var(--wp--preset--spacing--20, gb.$radius-medium);
+		font-weight: gb.$font-weight-medium;
 		white-space: nowrap;
 		margin: 0 8px;
+		font-family: gb.$default-font;
```

Comment:
This line was the primary improvement for the font; rest are just better use of attributes.